bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -406,14 +407,16 @@ export default {
         this.$f7.preloader.hide()
         this.$f7.dialog.alert('Error while signing out: ' + err)
       })
+    },
+    updateThemeOptions () {
+      this.themeOptions.dark = localStorage.getItem('openhab.ui:theme.dark') || (this.$f7.darkTheme ? 'dark' : 'light')
+      this.themeOptions.bars = localStorage.getItem('openhab.ui:theme.bars') || ((!this.$theme.ios) || (!this.$f7.darkTheme) ? 'filled' : 'light')
```

Comment:
Whoops that was actually if not iOS AND not dark then filled (I'm getting filled bars in dark mode by default on my desktop, this is a rather ugly combination). No worries I'll fix it with the next commit.